terraform/graph_walk.go

```diff
@@ -12,10 +12,9 @@ type GraphWalker interface {
 	EvalContext() EvalContext
 	EnterPath(addrs.ModuleInstance) EvalContext
 	ExitPath(addrs.ModuleInstance)
-	EnterVertex(dag.Vertex)
```

Comment:
The only implementation I could find of these functions was in the NullGraphWalker, so I removed them while I was in here.